File: backendOrderly/src/main/java/com/yebur/backendorderly/dto/output/OrderDetailResponse.java
package com.yebur.backendorderly.dto.output;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class OrderDetailResponse {

    private Long id;
    
    private Long productId;

    private Long orderId;

    private String comment;
    
    private int amount;

    private double unitPrice;
}

File: frontend/OrderlyTablet/app/src/main/java/com/example/orderlytablet/response/OrderDetailResponse.kt
package com.example.orderlytablet.response

data class OrderDetailResponse(
    val id: Long,
    val productId: Long,
    val productName: String,
    val orderId: Long,
    val comment: String,
    val amount: Int,
    val unitPrice: Double,
    val status: String,
    val paymentMethod: String
){
}